notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb

```diff
@@ -8,12 +8,12 @@
    },
```

Comment:
<p>the same comment about more info about API usage and parameters like in data2vec notebook need to be added</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>